File: front/src/app/Jogo.ts
import { Plataforma } from "./Plataforma";
import { Desenvolvedor } from "./Desenvolvedor";
import { Genero } from "./Genero";

export class Jogo {
    idJogo: number = 0;
    plataformaId: number = 0;
    plataforma: Plataforma | undefined;
    desenvolvedorId: number = 0;
    desenvolvedor: Desenvolvedor | undefined;
    generoId: number = 0;
    genero: Genero | undefined;
    nome: string = '';
  }

File: review-jogos-steam/Models/Jogo.cs
using System.ComponentModel.DataAnnotations;
namespace review_jogos_steam.Models;
using review_jogos_steam.Models;

public class Jogo
{
    [Key]
    public int Id { get; set; }

    public int? PlataformaId { get; set; }
    public Plataforma? Plataforma {get;  set;}

    public int? DesenvolvedorId { get; set; }
    public Desenvolvedor? Desenvolvedor { get; set; }

    public int? GeneroId { get; set; }
    public Genero? Genero {get; set;}

    public List<Tag>? Tags{get;  set;}
    public List<Avaliacao>? Avaliacoes {get;  set;}
    public List<Conquista>? Conquistas { get; set; }
    public string? Nome{get; set;}

}